Tools/cases_generator/generate_cases.py

```diff
@@ -381,7 +381,7 @@ def write_metadata(self, metadata_filename: str, pymetadata_filename: str) -> No
             self.write_pseudo_instrs()
 
             self.out.emit("")
-            self.write_uop_items(lambda name, counter: f"#define {name} {counter}")
+            self.out.emit('#include "pycore_uop_ids.h"')
```

Comment:
This creates dead code (`write_uop_items` is never called). I suppose at the end you're planning a grand removal of all of generate_cases.py?

Reply:
Yes